chain-impl-mockchain/src/ledger/evm.rs

```diff
@@ -42,6 +42,16 @@ pub struct AddressMapping {
     jor_to_evm: Hamt<DefaultHasher, JorAddress, EvmAddress>,
 }
 
+/// One way transforming procedure from `EvmAddress` to `JorAddress`.
+/// This allows to map any `EvmAddress` to the `JorAddress` before explicit execution of the `EvmMapping` transaction.
+/// Intention - is to have possibility to map EVM Contarct accounts with the Jormungandr accounts.
```

Comment:
This is confusing though, contract accounts should never be part of a `EvmMapping` transaction AFAIK, right?

Reply:
Yes, contract account should be a part of the `EvmMapping` certificate, will try to paraphrase it. I meant that with the usage of this function we can link contract account with the jormungandr.

Reply:
> contract account should be a part of the EvmMapping certificate

How can this happen?

Reply:
Sorry, mistaken 😓
> contract account should **NOT** be a part of the EvmMapping certificate